Nodejs/Product/Nodejs/Telemetry/NodejsTelemetryExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿//*********************************************************//
+//    Copyright (c) Microsoft. All rights reserved.
+//    
+//    Apache 2.0 License
+//    
+//    You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//    
+//    Unless required by applicable law or agreed to in writing, software 
+//    distributed under the License is distributed on an "AS IS" BASIS, 
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or 
+//    implied. See the License for the specific language governing 
+//    permissions and limitations under the License.
+//
+//*********************************************************//
+
+using System;
+
+namespace Microsoft.NodejsTools.Telemetry {
+    /// <summary>
+    /// Extensions for logging telemetry events.
+    /// </summary>
+    internal static class NodejsTelemetryExtensions {
+        public static void LogIntelliSenseActivatedForProject(this ITelemetryLogger logger, Guid projectGuid, Options.AnalysisLevel newAnalysisLevel) {
+            logger.ReportEvent(
+                TelemetryEvents.IntelliSenseActivatedForProject,
+                new DataPointCollection(
+                    new DataPoint(TelemetryProperties.ProjectGuid, projectGuid.ToString("B")),
+                    new DataPoint(TelemetryProperties.IntelliSenseLevel, newAnalysisLevel.ToString())));
+        }
+
+        public static void LogIntelliSenseLevelChanged(this ITelemetryLogger logger, Options.AnalysisLevel newAnalysisLevel) {
+            logger.ReportEvent(
+                TelemetryEvents.IntelliSenseLevelChanged,
+                TelemetryProperties.IntelliSenseLevel,
+                newAnalysisLevel.ToString());
+        }
```

Comment:
More generally, how did you test this? Also once this is in, we should add our query to the dashboard so that we can verify that we're not missing anything important.